instance.tf

```diff
@@ -0,0 +1,84 @@
+# Public EC2
+
+resource "aws_instance" "public" {
+  ami                         = "ami-090fa75af13c156b4"
+  instance_type               = "t2.micro"
+  associate_public_ip_address = true
+  subnet_id                   = aws_subnet.public.0.id
+  key_name                    = "moz"
+  vpc_security_group_ids      = [aws_security_group.public.id]
+
+  tags = {
+    Name = "${var.env_code}-Public"
+  }
+}
+
+# Private EC2
+
+resource "aws_instance" "private" {
+  ami                    = "ami-090fa75af13c156b4"
+  instance_type          = "t2.micro"
+  subnet_id              = aws_subnet.private.0.id
+  key_name               = "moz"
```

Comment:
Please change the key name to `main` it will be easier for me to troubleshoot  (if needed later in the course)

Reply:
will do